### ISSUE TITLE
Expand/collapse toggle not picked up by screen reader

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#303]
for the pivot facet (classification) - Call Number facet

_from accessibility testing with Michelle on 5/14/19_

**Comment by ajkiessl** [comment]
This works with the ChromeVox screen reader on Google Chrome.  We'll see if it causes issues tomorrow when testing with Michelle.

**Comment by ajkiessl** [comment]
I think we can close this.  @nicktiberi Did you and Michelle run into any issues with the facets while testing?

**Comment by nicktiberi** [comment]
Sorry, @ajkiessl — we forgot about this when testing with Michelle on Friday. 🤦 I think it is a pretty safe bet that we can close this, but maybe @binkylush can check with her to make sure this works properly and then we can close it.

**Comment by binkylush** [comment]
@ajkiessl and @nicktiberi - from Michelle's screen reader testing - "This looks like it works like it should.  Thank you."  So we're all set!
